front/app/containers/BulkEditPage/BulkEditView.tsx

```diff
@@ -122,6 +123,10 @@ class BulkEditView extends React.Component<BulkEditProps, BulkEditState> {
                     const isToRemove =
                       groupedByLabel.toRemove[label] &&
                       groupedByLabel.toRemove[label].includes(value);
+                    if (bucketKeyStringIsMissing(value)) {
```

Comment:
there are going to be cases where we actually will want to operate over `keyAsString`, but it gets ignored here. cases for that would be dates, where `keyAsString` is a much better way to reason over the aggs than the native key is.